libass/ass.c

```diff
@@ -574,29 +707,17 @@ static int process_style(ASS_Track *track, char *str)
     if (strcmp(target->Name, "Default") == 0)
         track->default_style = sid;
     return 0;
-
 }
 
-static bool format_line_compare(const char *fmt1, const char *fmt2)
+static bool format_line_compare(const ASS_FormatToken *fmt1, const ASS_FormatToken *fmt2, size_t num)
```

Comment:
Isn't it simply `memcmp()`?

Reply:
This function has fully rewritten anyway, I don't see much sense in rewriting it to only delete later.